by-language/java-jooq/src/generated/java/io/crate/demo/jooq/model/DefaultCatalog.java

```diff
@@ -17,13 +15,6 @@
 /**
  * This class is generated by jOOQ.
  */
-@Generated(
```

Comment:
I think having the code generated is ok as long as it's possible to do so out of the box.

Reply:
As the description stated, all these files are generated now. For whatever reason these comments don't get generated anymore.

Reply:
ok, i did not know this.